README.md

```diff
@@ -34,7 +34,7 @@ Push the operator bundle to your quay application repository as follows:
 operator-courier push deploy/catalog_resources/courier/0.1.0 kiegroup kogitocloud-operator 0.1.0 "basic XXXXXXXXX"
 ```
 
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. Also note that the push command does not overwrite an existing repository, and it needs to be deleted before a new version can be built and uploaded. Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) to load your operator bundle in OpenShift.
+If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. Notice that the push command does not overwrite an existing repository, and it needs to be deleted before a new version can be built and uploaded. Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) to load your operator bundle in OpenShift.
```

Comment:
WDYT about have a public quay.io repository in the org of this project with its image publish there by default? Then this explanation is no longer required. E.g https://quay.io/repository/aerogear/mobile-security-service-operator?tag=latest&tab=tags

Also, WDYT about manage the images by the CI? See an example [here](https://github.com/aerogear/mobile-security-service-operator/blob/master/.circleci/config.yml) and how the doc description over how the release tags are made [here](https://github.com/aerogear/mobile-security-service-operator#tags-release)

Reply:
Thanks! The CI thing is tracked and I should implement once I have the time!! Again, using your review as a input for a new JIRA.

BTW, we do have an organization with the Operator deployed. Those instructions is for someone interested in uploading to their own repo. We're going to upload to Operatorhub as well.

Reply:
https://issues.jboss.org/browse/KOGITO-146